go.mod

```diff
@@ -6,16 +6,17 @@ require (
 	contrib.go.opencensus.io/exporter/jaeger v0.2.0
 	contrib.go.opencensus.io/exporter/ocagent v0.6.0
 	contrib.go.opencensus.io/exporter/zipkin v0.1.1
-	github.com/UnnoTed/fileb0x v1.1.4 // indirect
```

Comment:
I think fileb0x should stay in the go.mod file.
It is used there: https://github.com/owncloud/ocis-konnectd/blob/105edd6195b7da588ec716c8a2c9a17fd4c9d062/pkg/assets/assets.go

Reply:
👍 yup, create a `tools.go` file and add a blank import for it @IljaN just like we do [here](https://github.com/owncloud/ocis-accounts/blob/master/tools.go)

Reply:
Thanks! Done.